plugin/cache/metrics.go

```diff
@@ -14,47 +14,54 @@ var (
 		Subsystem: "cache",
 		Name:      "entries",
 		Help:      "The number of elements in the cache.",
-	}, []string{"server", "type"})
+	}, []string{"server", "type", "zones"})
+	// cacheRequests is a counter of all requests through the cache.
+	cacheRequests = promauto.NewCounterVec(prometheus.CounterOpts{
+		Namespace: plugin.Namespace,
+		Subsystem: "cache",
+		Name:      "requests_total",
+		Help:      "The count of cache requests.",
+	}, []string{"server", "zones"})
 	// cacheHits is counter of cache hits by cache type.
 	cacheHits = promauto.NewCounterVec(prometheus.CounterOpts{
 		Namespace: plugin.Namespace,
 		Subsystem: "cache",
 		Name:      "hits_total",
 		Help:      "The count of cache hits.",
-	}, []string{"server", "type"})
-	// cacheMisses is the counter of cache misses.
+	}, []string{"server", "type", "zones"})
+	// cacheMisses is the counter of cache misses. - Deprecated
```

Comment:
Did you take the initiative to deprecate this, or did you get that from somewhere else?

Reply:
OK, I see that it is already deprecated in the upstream repository.

Reply:
oh yeah sorry, I just checked out these files from my branch that I made against upstream – this change came with it.